### ISSUE TITLE
[Feature Request] Raid Button

### ISSUE THREAD

**Issue by Toadfield** [night/betterttv#4567]
A Raid Button that works similar to the "Host Button" function.
Would save me some time.

**Comment by vanafanta** [comment]
YES! this would be amazing! so many times i am already on someones channel cuz of a colab and i can just hit raid instead of copy pasting the username going into my chat and then pressing raid. sometimes i like to browse the people im following and check out the stream before i actually raid so this would be HELLA beneficial

**Comment by Birdyvee** [comment]
YES! this would be amazing! so many times i am already on someones channel cuz of a colab and i can just hit raid instead of copy pasting the username going into my chat and then pressing raid. sometimes i like to browse the people im following and check out the stream before i actually raid so this would be HELLA beneficial
